CHANGELOG.md

```diff
@@ -1,22 +1,49 @@
 # Upcoming
 
+### ⚠️ Breaking Changes
+- Default `UITableView.loadingCell` changed:
```

Comment:
Can we keep the old functions and mark them as deprecated?
```
#deprecated(...., "Use func loadingCell(at indexPath:tectColor:) instead")
func loadingCell(at indexPath: IndexPath) -> UITableViewCell { ...
```
for all the stuff we're going to chance?

Reply:
I will do next time, but this is really small changes. Mostly for internal usage.
For now, we are going to make a new 1.6.0 version.